meta-ov/recipes-apps/ovmenu-ng/files/ovmenu-ng.sh

```diff
@@ -45,19 +45,23 @@ function submenu_file() {
 	--begin 3 4 \
 	--menu "You can use the UP/DOWN arrow keys" 15 50 4 \
 	Download_IGC   "Download XCSoar IGC files to USB" \
-	Download   "Download XCSoar to USB" \
-	Upload   "Upload files from USB to XCSoar" \
 	Update_Maps   "Update Maps" \
+	Upload_XCSoar   "Update or upload XCSoar files" \
+	Backup  "Backup XCSoar and OV settings" \
+	Restore   "Restore XCSoar and OV settings" \
+	Restore_XCSoar   "Restore only XCSoar settings" \
```

Comment:
I fail to grasp this diff. This removes 2 menu items and adds 4 different menu items. Then I read the commit message, but none of this appears to makes sense.